superset-frontend/src/SqlLab/components/ResultSet/ResultSet.test.jsx

```diff
@@ -105,51 +105,51 @@ test('renders a Table', () => {
   expect(wrapper.find(FilterableTable)).toExist();
 });
 
-describe('componentDidMount', () => {
```

Comment:
@lyndsiWilliams How should I rewrite these tests? or just remove it?

Reply:
You can test the functionality that was done within these tests, they were converted to functional but still live on these lines: https://github.com/apache/superset/blob/4ec10c23014cb9e5fe2cedf6407bb711cf05dbb8/superset-frontend/src/SqlLab/components/ResultSet/index.tsx#L192-L227